taskManager/Notifier.py

```diff
@@ -47,7 +47,47 @@ def connect_to_server(self):
                                              "gmail with --source_email and --source_password.")
         return server
 
-    def generate_message(self, subject, body, subject_prefix=True, attachment=None):
+    def parse_attachment_argument(self, arg):
+        """
```

Comment:
* rename function to something good
* include param arg description
* take out of class and put into a utils file
* use same function in both classes